### PR TITLE
Improve docs k0sctl airgap installation config example

### LINKED ISSUE #1616: Improve the k0sctl airgap documentation
There are some confusing parameters that could benefit from comments in the example yamls.

### PR DESCRIPTION
**Pull request by kke**
Signed-off-by: Kimmo Lehto <klehto@mirantis.com>

## Description

Improves the formatting and adds descriptive comments for the k0sctl airgap configuration example.

Fixes #1616 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## How Has This Been Tested?

- [ ] Manual test
- [ ] Auto test added

## Checklist:

- [ ] My code follows the style [guidelines](https://github.com/k0sproject/k0s/blob/main/docs/contributors/overview.md) of this project 
- [X] My commit messages are [signed-off](https://github.com/k0sproject/k0s/blob/main/docs/contributors/github_workflow.md)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by makhov — APPROVED** [review]

**Comment by kke** [comment]
Should look approximately like this:

image

File: docs/airgap-install.md [hunk 1]
```diff
@@ -57,34 +57,45 @@ Copy the `bundle_file` you created in the previous step or downloaded from the [
...
     version: 1.21.3+k0s.0
+  hosts:
+    - role: controller
+      ssh:
+        address: <controller-ip-address>
+        user: ubuntu
+        keyPath: /path/.ssh/id_rsa
+
+      #  uploadBinary: <boolean>
+      #    When true the k0s binaries are cached and uploaded
```
Comment (twz123) [inline docs/airgap-install.md:76]: Somehow I'm used to see the comments about settings above them, not below them.

**Review by twz123 — APPROVED** [review]

File: docs/airgap-install.md [hunk 2]
```diff
@@ -57,34 +57,45 @@ Copy the `bundle_file` you created in the previous step or downloaded from the [
...
-    k0sBinaryPath: /path/to/k0s_binary/k0s
-  - ssh:
-      address: <ip-address-worker>
-      user: ubuntu
-      keyPath: /path/.ssh/id_rsa
-    role: worker
-    uploadBinary: true
-    k0sBinaryPath: /path/to/k0s_binary/k0s
-    files:
-      - name: bundle-file
```
Comment (kke) [inline docs/airgap-install.md:82]: The name param is only used to display a custom name in the terminal output, it's a bit useless.

File: docs/airgap-install.md (hunk 1 above)
Comment (kke) [reply docs/airgap-install.md:76]: Yeah the first row was intended to be some kind of title for the doc block, not a template. This is more apparent in the comment for the k0sBinaryPath where the "template" has been commented out.

**Pull request merged**